src/MailCollector.php

```diff
@@ -427,10 +420,11 @@ public function displayFoldersList($input_id = "")
      */
     private function displayFolder($folder, $input_id)
     {
-        $fname = mb_convert_encoding($folder->getLocalName(), "UTF-8", "UTF7-IMAP");
+        $fglobalname = mb_convert_encoding($folder->getGlobalName(), "UTF-8", "UTF7-IMAP");
+        $flocalname  = mb_convert_encoding($folder->getLocalName(), "UTF-8", "UTF7-IMAP");
```

Comment:
To prevent issues with quotes in folder names.
```suggestion
        $fglobalname = htmlspecialchars(mb_convert_encoding($folder->getGlobalName(), "UTF-8", "UTF7-IMAP"), ENT_QUOTES);
        $flocalname  = htmlspecialchars(mb_convert_encoding($folder->getLocalName(), "UTF-8", "UTF7-IMAP"), ENT_QUOTES);
```